velox/functions/prestosql/DecimalVectorFunctions.cpp

```diff
@@ -454,9 +511,16 @@ std::vector<std::shared_ptr<exec::FunctionSignature>> decimalRoundSignature() {
           .integerVariable("a_scale")
           .integerVariable(
               "r_precision", "min(38, a_precision - a_scale + min(1, a_scale))")
-          .integerVariable("r_scale", "0")
-          .returnType("DECIMAL(r_precision, r_scale)")
-          .argumentType("DECIMAL(a_precision, a_scale)")
+          .returnType("decimal(r_precision, 0)")
```

Comment:
`DECIMAL` still needs to be upper base (we will probably fix it later, but still need this hack for now).